stable/kibana/templates/deployment.yaml

```diff
@@ -54,6 +54,12 @@ spec:
         #     port: {{ .Values.service.internalPort }}
         resources:
 {{ toYaml .Values.resources | indent 10 }}
+        volumeMounts:
+        {{- range $configFile := (keys .Values.configFiles) }}
+        - name: {{ template "kibana.name" $ }}
+          mountPath: "/usr/share/kibana/config/{{ $configFile }}"
```

Comment:
By default kibana stores its keys in the config folder. If we are leveraging this system, we should probably make a note in the README that if anyone was leveraging a custom build with inbuilt certs they should not place those in this directory but rather in a different directory and leverage the below fields in the kibana.yml

```
server.ssl.enabled: true
server.ssl.key: /path/to/your/server.key
server.ssl.certificate: /path/to/your/server.crt
```

Reply:
Since `subPath` is being used here, it won't affect any other files in the directory except for those provided through the `configFiles` variable in values.yaml. I have now added this information to the readme too.